recipes/scijava-jupyter-kernel/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -euo
+
+# Install Beakerx...
+# ...waiting for a Maven artifact.
+cd $RECIPE_DIR/
+
+git clone https://github.com/twosigma/beakerx.git
+cd beakerx/
+git checkout 585f07c5dfe7f9f0f97053d90ffc5a696d972382
```

Comment:
> Hope you understand.

If you add some comments in the recipe mentioning that and open an issue in the feedstock to track that I don't see any major blocks. But we need to document this decision to avoid getting lost in the process.